relay_thegraph.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
could you extend the default config and just change the schema values? so we don't have to keep two in sync in the future